src/handler.rs

```diff
@@ -240,6 +240,15 @@ impl Call {
         }
     }
 
+    /// Returns `id` of the channel, if connected to any.
```

Comment:
```suggestion
    /// Returns `id` of the channel, if connected to any.
    ///
    /// **Note**: The returned `id` will not correctly update if a connected bot
    /// is moved between channels manually. This will be fixed in the next
    // breaking release.
```

Reply:
I expanded a bit your suggestion. Check it out